config/quota-file.json

```diff
@@ -0,0 +1,50 @@
+{
+  "@context": "https://linkedsoftwaredependencies.org/bundles/npm/@solid/community-server/^2.0.0/components/context.jsonld",
+  "import": [
+    "files-scs:config/app/main/default.json",
+    "files-scs:config/app/init/default.json",
+    "files-scs:config/app/setup/required.json",
+    "files-scs:config/http/handler/default.json",
+    "files-scs:config/http/middleware/websockets.json",
+    "files-scs:config/http/server-factory/websockets.json",
+    "files-scs:config/http/static/default.json",
+    "files-scs:config/identity/access/public.json",
+    "files-scs:config/identity/email/default.json",
+    "files-scs:config/identity/handler/default.json",
+    "files-scs:config/identity/ownership/token.json",
+    "files-scs:config/identity/pod/static.json",
+    "files-scs:config/identity/registration/enabled.json",
+    "files-scs:config/ldp/authentication/dpop-bearer.json",
+    "files-scs:config/ldp/authorization/allow-all.json",
+    "files-scs:config/ldp/handler/default.json",
+    "files-scs:config/ldp/metadata-parser/default.json",
+    "files-scs:config/ldp/metadata-writer/default.json",
+    "files-scs:config/ldp/modes/default.json",
+    "files-scs:config/storage/backend/pod-quota-file.json",
+    "files-scs:config/storage/key-value/resource-store.json",
+    "files-scs:config/storage/middleware/default.json",
+    "files-scs:config/util/auxiliary/acl.json",
+    "files-scs:config/util/identifiers/suffix.json",
+    "files-scs:config/util/index/default.json",
+    "files-scs:config/util/logging/winston.json",
+    "files-scs:config/util/representation-conversion/default.json",
+    "files-scs:config/util/resource-locker/memory.json",
+    "files-scs:config/util/variables/default.json"
+  ],
+  "@graph": [
+    {
+      "comment": "A single-pod server that stores its resources on disk while enforcing quota."
+    },
+    {
+      "@id": "urn:solid-server:default:QuotaStrategy",
+      "PodQuotaStrategy:_limit_amount": 7000,
+      "PodQuotaStrategy:_limit_unit": "bytes",
+      "GlobalQuotaStrategy:_limit_amount": 10000,
+      "GlobalQuotaStrategy:_limit_unit": "bytes"
+    },
+    {
+      "@id": "urn:solid-server:default:SizeReporter",
+      "FileSizeReporter:_ignoreFolders": [ "^/\\.internal$" ]
+    }
```

Comment:
At the time it seemed logical to me that the server's internal files wouldn't count towards the quota.

But in the end this is up to the user/admin. Lmk if you want me to remove it.

Having this in the basic config might make it easier for users/admins to find how to configure this parameter.

Reply:
My point here is that this config is importing `files-scs:config/storage/backend/pod-quota-file.json`, which sets up the server with a pod quota system. So those `GlobalQuotaStrategy` parameters are irrelevant here. The `ignoreFolders` parameter also seems irrelevant since that one has no impact on pod strategy, but perhaps that one can stay indeed for clarity.